demos/list.html

```diff
@@ -1200,6 +1329,26 @@ <h3>Example - Interactive List (Dark Theme)</h3>
             evt.preventDefault();
           });
         }
+
+        function clickHandler(evt, li) {
+          var checkbox = li.querySelector('.mdc-checkbox__native-control');
+          if(evt.target === li) {
+            checkbox.checked = !checkbox.checked;
+          }
+        };
+
+        function addRippleToListItems(list) {
```

Comment:
You should also forEach over `list.querySelectorAll('.mdc-checkbox')` to instantiate checkboxes (which will instantiate ripples on them) to benefit from #2123.

Reply:
This is happening on line #1309. I wanted to group all the checkbox logic together.